en/docs/connectors/catalog/built-in/graphql/example.md

```
@@ -209,19 +209,19 @@ Navigate back to **GraphQL Service - /graphql** in the left tree. The GraphQL di
 1. Select **Run Integration** (▶) in the editor toolbar. The integration starts and logs appear in the output panel.
 2. Send a GraphQL query to the running service using one of the following methods:
 
-   **Option 1 — WSO2 Integrator built-in HTTP client:** Use the built-in HTTP client or **Try It** panel in WSO2 Integrator to POST a GraphQL query to `/graphql`.
+   **Option 1: WSO2 Integrator built-in HTTP client:** Use the built-in HTTP client or **Try It** panel in WSO2 Integrator to POST a GraphQL query to `/graphql`.
 
-   **Option 2 — curl:** Run the following command in a terminal, replacing the port with the value you set for `graphqlPort`:
+   **Option 2: curl:** Run the following command in a terminal, replacing the port with the value you set for `graphqlPort`:
 
    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)